src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -579,24 +587,18 @@ private void EnterParameters(ImmutableArray<ParameterSymbol> parameters)
         private void EnterParameter(ParameterSymbol parameter)
         {
             int slot = GetOrCreateSlot(parameter);
-            if (parameter.RefKind == RefKind.Out && !this._currentMethodOrLambda.IsAsync) // out parameters not allowed in async
-            {
-            }
-            else
+            Debug.Assert(!IsConditionalState);
+            if (slot > 0 && parameter.RefKind != RefKind.Out)
             {
-                Debug.Assert(!IsConditionalState);
-                if (slot > 0 && parameter.RefKind != RefKind.Out)
+                var paramType = parameter.Type.TypeSymbol;
+                if (EmptyStructTypeCache.IsTrackableStructType(paramType))
                 {
-                    var paramType = parameter.Type.TypeSymbol;
-                    if (EmptyStructTypeCache.IsTrackableStructType(paramType))
-                    {
-                        InheritNullableStateOfTrackableStruct(paramType, slot, valueSlot: -1, isByRefTarget: parameter.RefKind != RefKind.None, slotWatermark: GetSlotWatermark());
-                    }
+                    InheritNullableStateOfTrackableStruct(paramType, slot, valueSlot: -1, isByRefTarget: parameter.RefKind != RefKind.None, slotWatermark: GetSlotWatermark());
                 }
             }
         }
 
-#region Visitors
+        #region Visitors
```

Comment:
Nit: indent

Reply:
Not sure what causes the `#region` indent to change but this seems to change back and forth.